tests/integration/test_duckdb_tpch.py

```diff
@@ -0,0 +1,89 @@
+import duckdb
+import pytest
+
+from ..consumers.duckdb_consumer import DuckDBConsumer
+from ..parametrization import custom_parametrization
+from ..verification import verify_equals
+from .queries.tpch_test_cases import TPCH_QUERY_TESTS
+
+
+@pytest.mark.usefixtures("prepare_tpch_parquet_data")
+class TestDuckDBConsumer:
+    """
+    Test Class for testing Substrait using DuckDB as a consumer.
+    """
+
+    @staticmethod
+    @pytest.fixture(scope="class", autouse=True)
+    def setup_teardown_class(request):
+        cls = request.cls
+
+        cls.db_connection = duckdb.connect()
+        cls.db_connection.execute("INSTALL substrait")
+        cls.db_connection.execute("LOAD substrait")
+        cls.consumer = DuckDBConsumer(cls.db_connection)
+        cls.created_tables = set()
+
+        yield
```

Comment:
What is this yield here for?

Reply:
the yield is for the `setup_teardown_class` fixture.

Once all the tests are run, anything after this yield is called.  In this specific case, we close the duckdb connection at the end.